src/browser/services/canny.ts

```diff
@@ -54,7 +57,7 @@ export const CannySDK = {
           return
         }
         const f = document.getElementsByTagName('script')[0]
-        const e = document.createElement('script') as HTMLScriptElement
+        const e = document.createElement('script')
```

Comment:
It's not clear to me that the check for an element with id `canny-jssdk` will definitely prevent the script from being added more than once. We should definitely check this before merging this PR... :-)

Reply:
I think you're right, and it seems it'd load again even if not needed from testing. However I feel it's nigh impossible to a perfect cleanup and remount of the external script from canny. The worst case of an extra load from canny is still fine I think